exec/java-exec/src/main/java/org/apache/drill/exec/planner/ParquetPartitionDescriptor.java

```diff
@@ -130,13 +131,13 @@ protected void createPartitionSublists() {
   }
 
   @Override
-  public TableScan createTableScan(List<PartitionLocation> newPartitionLocation) throws Exception {
+  public TableScan createTableScan(List<PartitionLocation> newPartitionLocation, String cacheFileRoot) throws Exception {
```

Comment:
ParquetPartitionDescriptor does not supportsSinglePartOptimization. Should it here throws unsupportedException, and implement createTableScan(newPartitionLocation) only?